src/main/java/org/casbin/adapter/HibernateAdapter.java

```diff
@@ -59,10 +60,10 @@ private void open() throws SQLException {
     private void createDatabase() {
         Session session = factory.openSession();
         Transaction tx = session.beginTransaction();
-        if (this.databaseProductName.contains("MySQL")) {
+        if (this.databaseProductName.contains(AdapterConstants.MySQL.name()) || this.databaseProductName.contains(AdapterConstants.MariaDB.name())) {
```

Comment:
Don't repeat this thing so many times. Make it simpler and smaller

Reply:
thanks, added a new PR that is simpler